opendevin/runtime/plugins/agent_skills/setup.sh

```diff
@@ -11,3 +15,72 @@ echo 'export PYTHONPATH=/opendevin/plugins/agent_skills:$PYTHONPATH' >> ~/.bashr
 export PYTHONPATH=/opendevin/plugins/agent_skills:$PYTHONPATH
 
 pip install flake8 python-docx PyPDF2 python-pptx pylatexenc openai opencv-python
+
+# ======================
+# Jupyter
+# ======================
+source ~/.bashrc
+# if user name is `opendevin`, add '/home/opendevin/.local/bin' to PATH
+if [ "$USER" = "opendevin" ]; then
+    echo 'export PATH=$PATH:/home/opendevin/.local/bin' >> ~/.bashrc
+    export PATH=$PATH:/home/opendevin/.local/bin
```

Comment:
Do we need to it twice? I guess maybe we can echo them into .bashrc then source it in the end.

Reply:
Sure! updated.